README.md

```diff
@@ -1,6 +1,8 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To run this project locally .env.development and .env.test files should be created in the root to link the code to the database: PGDATABASE=name_of_database_in_setup.sql
```

Comment:
Good start! I think we can be clearer here about which db name goes into while file - is there anything else the user should know in order to be able to get the server up and running locally? Just creating the env files won't actually run the server - anything else they need to do? (hint: how do they make sure they have all the packages installed locally and that the databases exist locally etc?)